starter-code/styles/modules.css

```diff
@@ -9,12 +9,23 @@
   padding: 0 0 .2em 1em;
 }
 
-.icon-menu {
-  /* TODO: Add styles for the icon-menu class (whether it be the
-      hamburger font icon or an image of your choosing). It should be
-      bigger, and off to the right. */
+@media (max-width: 960px) {
+  .icon-menu {
+    /* TODO:DONE: Add styles for the icon-menu class (whether it be the
+        hamburger font icon or an image of your choosing). It should be
+        bigger, and off to the right. */
+    position: fixed;
+    top: 0;
+    right: 0;
+    width: 150px;
+    height: 150px;
```

Comment:
To make your glyph icon larger, you can simply increase the font size, as it's just a font!